documentation/puppet-api/v3/environment_modules.markdown

```diff
@@ -16,10 +16,55 @@ on Ruby Puppet masters, such as the [deprecated WEBrick Puppet master][]. It als
 the Ruby-based Puppet master's authorization methods and configuration. See the
 [Authorization section](#authorization) for more information.
 
-## `GET /puppet/v3/environment_modules?environment=:environment`
+## `GET /puppet/v3/environment_modules`
```

Comment:
Ah yeah....I need to do that. Was going to save it in case the structure some how changed since it's a manual process to update that file.